statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -1,13 +1,15 @@
 package presenter
 
 import (
-	"math"
 	"math/big"
 	"strings"
 
 	"github.com/ElrondNetwork/elrond-go/core"
 )
 
+// denomination is equals with 10^-4
+var denominationCoefficient = big.NewFloat(0.0001)
```

Comment:
This should be in economics.toml for easier configuration
Should be named in a way as to suggest that is only used in displaying rewards earned

Reply:
Done.